website_cms/templates/misc.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <template id="website.publish_management">
+
+    <!-- we do not want additional dropdown for cms.page -->
+    <!--
+    this was supposed to work (tested w/ chrome) but looks like odoo does not accept it :S rewrite everything...
+      <xpath expr="//button[contains(@class, 'dropdown-toggle')]/following-sibling::ul" position="replace" />
```

Comment:
is not that simple and the only way to do this "safely" w/out touching odoo itself is to override it.
An option could be to patch and PR odoo template.